client/gutenberg/extensions/presets/jetpack/editor-shared/test/get-gutenberg-site-fragment.js

```diff
@@ -0,0 +1,52 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import getGutenbergSiteFragment from '../get-gutenberg-site-fragment';
+
+describe( 'getGutenbergSiteFragment()', () => {
+	let beforeWindow;
+
+	beforeAll( () => {
+		beforeWindow = global.window;
+		global.window = {
+			location: {},
+		};
+	} );
+
+	afterAll( () => {
+		global.window = beforeWindow;
+	} );
+
+	test( 'should return null by default', () => {
+		window.location.pathname = '/';
+		expect( getGutenbergSiteFragment() ).toBeNull();
+	} );
+
+	test( 'should return null when starting a new post in wp-admin', () => {
+		window.location.pathname = '/wp-admin/post.php';
```

Comment:
> should return null when starting a new post in wp-admin

this should be `post-new.php`

Reply:
👍 Done in https://github.com/Automattic/wp-calypso/pull/28998/commits/06e355600bb4502f0e07ecbcad52dbf299a0b05f